### DISCUSSION TITLE
Missing call base.OnPropertyChanged

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#7045] [Q&A] [answered]
In this lines is missing OnPropertyChanged.
https://github.com/AvaloniaUI/Avalonia/blob/e0bae82fdbd254a6bf692b3dca9e3971ba8eeb63/src/Avalonia.Controls/Window.cs#L995-L1014

Is it by design or issue?

**Comment by kekekeks** [comment, marked as answer]
The code was added in https://github.com/AvaloniaUI/Avalonia/commit/5d007606bad8a8d9509129f1b4f7e9d7d43aeadd and should be only concerned by `SystemDecorationsProperty`. So definitely not by design.
